pkg/machines/components/vm/vmExpandedContent.scss

```diff
@@ -55,6 +55,30 @@
     }
   }
 
+  .overview-tab .pf-c-description-list {
```

Comment:
I don't understand why this is needed here. 
Description list (by itself) works correctly by wrapping when in small screens.

Reply:
The labels are exceedingly long and shove the content to be way too wide, causing content to be shoved way offscreen.

The problem is that the description list is not by itself; it is side-by-side with another, and since they're both much larger (taller and, more importantly, wider) than when we worked on the PR originally, it breaks the page.

We still need to fix it for non-Firefox browsers (everything that does not _yet_ support subgrid), of course.

Reply:
Do you think it makes sense to have a global override (patternfly-4-overrides)  for the description list making it less wide then?

I have similar issue in other PR, https://github.com/cockpit-project/cockpit/pull/14754/commits/f0027071cb50b206c46d31b34df56481500a3393#diff-325ba8d07b7191456e22e2785c61be383372356c4e38b9a7cbfe35021be2af96R50

And it would be nice to keep it globally consistent.

Reply:
Yeah, I think so. 

We should try to upstream both, and meanwhile, apply (probably) both at a Cockpit-global override.

The only problem I really see is that — for the time being — only Firefox would have the optimized subgrid layout, so there would be two codepaths. It's not _that_ different, but it is something to consider. Hopefully Chromium and WebKit will _finally_ get subgrid in the next few months. (They're both supposed to.)

https://blog.chromium.org/2020/06/improving-chromiums-browser.html

> Chromium still doesn't support subgrid.
>
> Hopefully it won't be an exception for much longer. It's still early days, but I'm excited to share that a team at Microsoft Edge are working on rearchitecting Chromium's Grid support to use the new LayoutNG engine - and as part of this are intending to add subgrid support! We're excited to see this feature land and want to express our appreciation to our partners at Microsoft, Igalia, and the many other Chromium contributors who have worked on CSS Grid support.

And it's on WebKit's radar: https://bugs.webkit.org/show_bug.cgi?id=202115